src/einsteinpy/coordinates/__init__.py

```diff
@@ -0,0 +1,7 @@
+from einsteinpy.coordinates.core import BoyerLindquist, Cartesian, Spherical
+
+from einsteinpy.coordinates.velocity import (  # isort:skip
```

Comment:
Can you add a TODO here? Referencing the issue. And create an issue in EInsteinPy too! With link to the issue in isort

Reply:
```TODO``` keyword was causing CodeFactor to fail. Added a descriptive comment instead.